crates/holochain_conductor_api/src/config/conductor/dpki_config.rs

```diff
@@ -77,11 +79,17 @@ impl DpkiConfig {
 
 impl Default for DpkiConfig {
     fn default() -> Self {
-        Self {
-            dna_path: None,
-            network_seed: DPKI_NETWORK_SEED_TESTING.to_string(),
-            allow_throwaway_random_dpki_agent_key: false,
-            no_dpki: false,
+        cfg_if! {
+            if #[cfg(feature = "dpki")] {
```

Comment:
We should keep testing with DPKI enabled until that becomes a problem, if it ever does, and we can discuss then what to do about that. This would need to go into `DEFAULT_FEATURES` in the Makefile and `.config/test-args.nix`